recipes/conda-devenv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "conda-devenv" %}
+{% set version = "0.9.0" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: https://github.com/ESSS/conda-devenv.git
```

Comment:
There's a tag on the repository, but we don't pretend to release to PyPI since it is meant to be used with conda only (like `conda-smithy` itself for example).